spec/models/food_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
```

Comment:
Well done with all the unit and integration specs!!! 🚀 

![image](https://github.com/olanikegloria/Recipe_app/assets/38088818/59841042-4ac0-40b9-aa2c-9d9d0e711eed)